app/containers/List/ListPicker.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Text } from 'react-native';
+
+import { TActionSheetOptionsItem, useActionSheet } from '../ActionSheet';
+import { CustomIcon } from '../CustomIcon';
+import * as List from './index';
+import I18n from '../../i18n';
+import { useTheme } from '../../theme';
+import sharedStyles from '../../views/Styles';
+
+type TOption = { label: string; value: string; second?: string };
```

Comment:
What's `second`?
Can you try a more semantic naming?

Edit: After reading through your PR, is something I already commented.
This component is supposed to be generic instead of couple to a logic.
Why would a `list picker` understand what's a `second` of a notification option?